firefox_ui_tests/remote/security/test_safe_browsing_notification.py

```diff
@@ -56,11 +62,16 @@ def test_notification_bar(self):
 
                 # Navigate to the unsafe page
                 # Check "ignore warning" link then notification bar's "not badware" button
-                self.marionette.navigate(unsafe_page)
-                # Wait for the DOM to receive events for about:blocked
-                time.sleep(1)
-                self.check_ignore_warning_button(unsafe_page)
-                self.check_not_badware_button(label, report_page)
+                # Only do this if feature supports it
+                if label is not None:
+                    self.marionette.navigate(unsafe_page)
+                    # Wait for the DOM to receive events for about:blocked
+                    time.sleep(1)
+                    self.check_ignore_warning_button(unsafe_page)
```

Comment:
The "ignore warning" link gets tested for unwanted software on line 80, so it does not go untested. It cannot be tested here on line 70 because this part is used to test the report functionality, which unwanted software blocking does not have. So, I believe my logic is correct here.